src/Service.js

```diff
@@ -62,16 +62,17 @@ const post = async (path, body) => {
 //   }
 //   return data;
 // };
-// const remove = async (path) => {
-//   path = sanitizePath(path);
-//   const response = await fetch(baseUrl + path, requestOptionsBuilder('DELETE'));
-//   const data = await response.json();
-//   if (response.status >= 400 && response.status < 600) {
-//     return Promise.reject(response);
-//   }
-//   return data;
-// };
+const remove = async (path) => {
+  path = sanitizePath(path);
+  const response = await fetch(baseUrl + path, requestOptionsBuilder('DELETE'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8494981)
```suggestion
  const response = await fetch(baseUrl + path, requestOptionsBuilder("DELETE"));
```